src/main/java/sdmExam/RepeatedPlayException.java

```diff
@@ -0,0 +1,8 @@
+package sdmExam;
+
+public class RepeatedPlayException extends Exception{
+    public RepeatedPlayException(String player_colour){
+
+        super("A player" + player_colour + " cannot play twice in a row.");
```

Comment:
I don't think there we need to report also which is the player, we can just say that it is not possible to play twice in a row

Reply:
Therefore method `getPlayerColour` is not necessary

Reply:
I agree, I do it now

Reply:
I thought might be "cool" to report who tried the invalid action between players.

Reply:
Yea, but I do not think it's necessary, also your solution it's hardcoded. If it will be necessary, we will add it later